### DISCUSSION TITLE
connect ECONNREFUSED 127.0.0.1:5432

### DISCUSSION THREAD

**Discussion by SPbb0451** [akiver/cs-demo-manager#656] [Q&A] [answered]
I followed the documentation and installed it, but I get an error about not being able to connect!
I didn't find a solution in the FAQ.

**Comment by akiver** [comment, marked as answer]
Assuming you are on Windows, here are some things you can do:

1. Make sure the `postgresql` service is running
	1. Search for `services` from the Windows search bar and open the `Services` app
	2. Find the service `postgresql-x64-16` and ensure it's running. If it's missing, it means PostgreSQL has not been installed correctly.
	
![post](https://github.com/akiver/cs-demo-manager/assets/1842524/22fe00d4-b0d0-4386-a8cb-ba5319624ae9)

2. Make sure the port is correct. The default port is `5432`, maybe you changed it during the installation like on the screenshot in the installation instructions? (it's `5433`)
	1. Open a terminal (`cmd.exe`)
	2. Type `psql -h 127.0.0.1 -U postgres -p 5432` and press enter. If you have the "Connection refused" error try with the port `5433`.
4. Re-install PostgreSQL and make sure to set the port to `5432` during the installation

**Reply by SPbb0451** [reply]
Oh indeed, the service isn't working properly, thanks very much for the reply!

**Reply by akiver** [reply]
👍 I added a documentation https://cs-demo-manager.com/docs/faq/installation#econnrefused-error

**Comment by lekeshibai** [comment]
![Snipaste_2025-05-21_23-38-41](https://github.com/user-attachments/assets/2ca5b845-9133-4b45-907c-bc34971bc1a8)
![Snipaste_2025-05-21_23-39-01](https://github.com/user-attachments/assets/51b2d0c7-6b87-47de-bc10-92705dd20508)

An error occurred when connecting to the remote server
